src/js/ui/home.js

```diff
@@ -0,0 +1,26 @@
+import { capitalizeFirstLetter, getID } from '../utils.js';
+
+function createPokemonTile(pokemon, sprite) {
+    const $template = document.getElementById('tile-template').content.cloneNode(true);
+
+    const { name, url } = pokemon;
+    const ID = getID(url);
+    
+    $template.querySelector('p').textContent = capitalizeFirstLetter(name);
+    $template.querySelector('img').src = sprite(ID);
+    $template.querySelector('img').dataset.id = ID;
+    $template.querySelector('div').dataset.id = ID;
+    
+    return $template;
+}
+
+function appendPokemon($pokemon) {
+    const $list = document.getElementById('pokemons-list');
+    $list.appendChild($pokemon);
+}
+
+export function fillPage(pokemons, sprites) {
+    document.getElementById('pokemons-list').innerHTML = "";
```

Comment:
When you write something like "pokemons", "fruits", "cats" you kinda know that inside will be more than 1 of these things, that's why I recommend you avoid using list/array/something similar like: `const pokemonArray` or `const pokemonList`